apps/ai-image-generator/frontend/build-actions.js

```diff
@@ -58,7 +58,7 @@ const main = async (watch = false) => {
       platform: 'node',
       outdir: 'build',
       logLevel: 'info',
-      format: 'cjs',
+      format: 'esm',
```

Comment:
definitely want to keep an eye on this when we build for production, but if that helps it work locally then 👍